packages/in_app_purchase/in_app_purchase_ios/example/ios/RunnerTests/Stubs.m

```diff
@@ -261,7 +261,10 @@ @implementation FIAPReceiptManagerStub : FIAPReceiptManager
 
 - (NSData *)getReceiptData:(NSURL *)url error:(NSError **)error {
   if (self.returnError) {
-    *error = [[NSError alloc] init];
+    *error = [NSError
+        errorWithDomain:@"test"
+                   code:1
+               userInfo:@{@"name" : @"test", @"houseNr" : @5, @"error" : [[NSError alloc] init]}];
```

Comment:
Please add some information to this internal NSError and then a unit test that validates that you get that information. Right now, if someone removed this entry—the purpose of which is entirely non-obvious outside of the context of this PR, and people do not always go look at the PR where something was added when removing it—no tests would fail, we would just silently lose coverage.